scripts/installer/windows/scripts/stop_service.vbs

```diff
@@ -0,0 +1,3 @@
+DIM objshell
+set objshell = wscript.createobject("wscript.shell")
```

Comment:
why do we need this file?

Reply:
I add a custom step to stop promptflow service to mitigate the transient issue that windows installer close service failed. We need use vbs to avoid poping up a window.
I have tried to use quietexecmd, but it failed to detect command for unknown reason for now
![image](https://github.com/microsoft/promptflow/assets/26239730/9c18a3bd-b45b-4834-be2c-6b39f52384d8)